docs/components/oculus-go-controls.md

```diff
@@ -42,16 +42,27 @@ and/or pressed buttons (trackpad, trigger).
 
 | Event Name         | Description           |
 | ----------         | -----------           |
-| trackpadchanged    | Trackpad changed.     |
-| trackpaddown       | Trackpad pressed.     |
-| trackpadup         | Trackpad released.    |
-| trackpadtouchstart | Trackpad touched.     |
-| trackpadtouchend   | Trackpad not touched. |
-| trackpadmoved      | Trackpad moved.       |
+| touchpadchanged    | Touchpad changed.     |
+| touchpaddown       | Touchpad pressed.     |
+| touchpadup         | Touchpad released.    |
+| touchpadtouchstart | Touchpad touched.     |
+| touchpadtouchend   | Touchpad not touched. |
+| touchpadmoved      | Touchpad moved.       |
 | triggerchanged     | Trigger changed.      |
 | triggerdown        | Trigger pressed.      |
 | triggerup          | Trigger released.     |
 
+### Legacy WebVR Browsers
+
+Legacy (WebVR-based) Oculus Go browsers use the older 'trackpadXXXX' event names, rather than the
```

Comment:
We can say:
```Legacy WebVR browsers use the older 'trackpadXXXX' event names```

It's not exclusive to Oculus Go. It also applies to Firefox desktop for instance.

Reply:
I'll fix and push a new pull request. (I did not realize this pull make it to the aframe github; I intended it to be doc changes only.)